app/index.html

```diff
@@ -50,6 +50,8 @@
   <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
   <!-- endbuild -->
 
+<link rel="import" href="../../bower_components/polymer/polymer.html">
```

Comment:
You don't want to do this. It means the app will attempt to load Polymer twice (once in the imported bundle in elements.vulcanized.html and once here.